src/Apps/Sell/Utils/acceptableConditionsForSubmission.tsx

```diff
@@ -0,0 +1,15 @@
+import { ArtworkConditionEnumType } from "__generated__/useUpdateArtworkMutation.graphql"
+
+// This type is here to make sure the condition options match the allowed values in the schema. We have to update the conditionOptions array whenever the ArtworkConditionEnumType changes.
+type ConditionOption = {
+  text: string
+  value: Exclude<ArtworkConditionEnumType, "%future added value"> | ""
+}
+
+export const conditionOptions: ConditionOption[] = [
```

Comment:
What do you think about renaming this file to `conditionOptions.tsx` so it's easier to find? 🤔